R/launcher-functions.R

```diff
@@ -295,52 +295,56 @@ launcherSubmitJob <- function(name,
                   applyConfigSettings = applyConfigSettings)
 }
 
+#' Execute an R Script as a Launcher Job
+#'
+#' Convenience function for running an R script as a launcher job using whichever
+#' R is found on the path in the launcher cluster.
+#'
+#' See [launcherSubmitJob()] for running jobs with full control over command,
+#' environment, and so forth.
+#'
+#' @param script Fully qualified path of R script. Must be a path that is
+#'   available in the job container (if using containerized job cluster such as Kubernetes).
+#' @param cluster The name of the cluster this job should be submitted to.
+#' @param container The container to be used for launched jobs.
+#'
+#' @family job submission
+#' @export
+launcherSubmitR <- function(script,
+                            cluster = "Local",
+                            container = NULL)
+{
+  # don't proactively check for existence of the script; possible the supplied
+  # path only resolves correctly in the job cluster
+  scriptPath <- path.expand(script)
+  scriptFile <- basename(scriptPath)
+  scriptArg <- paste("-f", scriptPath)
+  jobTag <- paste("rstudio-r-script-job", scriptFile, sep = ":")
+  callLauncherFun("launcher.submitJob",
+                  args = c("--slave", "--no-save", "--no-restore", scriptArg),
+                  cluster = cluster,
+                  command = "R",
```

Comment:
Maybe make this a parameter so users could select a particular version of R to run with (e.g. if they had already configured their cluster machines to provide R at `/opt/local/R/3.5.2/bin/R` or something)?